tests/e2e/testsLibrary/CodeExecutionTests.ts

```diff
@@ -152,16 +164,21 @@ export class CodeExecutionTests {
         return CodeExecutionTests.lastApplicationUrl;
     }
 
+    /**
+     * @deprecated This method should not be used, preview widget is no longer a valid page fragment.
+     */
     public refreshPreviewWindow() {
         test('Refreshing preview widget', async () => {
-            await this.previewWidget.refreshPage();
+            Logger.warn(`Method is deprecated.`);
```

Comment:
"Deprecated" means still working, but to be removed soon.
This method looks like not working, so let's remove it at all then.

Reply:
@dmytro-ndp that would cause a lot of changes in devfile tests.
I wanted to avoid having to deal with them right now.
Full removal will be the next step

Reply:
In that case let's treat them as absolute, not deprecated.